modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/services/checker/excel/ExcelService.scala

```diff
@@ -71,7 +71,10 @@ class ExcelService {
   private def setCells(workbook: XSSFWorkbook, changeFields: List[ExcelMediaInformationChange]): Unit = {
     changeFields.foreach(f => {
       val sheet = workbook.getSheetAt(f.sheetIdx)
-      this.getCell(sheet, f.cell).setCellValue(f.newValue)
+      f.newValue.toDoubleOption match {
+        case Some(v) => this.getCell(sheet, f.cell).setCellValue(v)
+        case _ => this.getCell(sheet, f.cell).setCellValue(f.newValue)
+      }
```

Comment:
```suggestion
      val newValue = f.newValue.toDoubleOption match {
        case Some(v) => v
        case _ => f.newValue
      }
      this.getCell(sheet, f.cell).setCellValue(newValue)
```

Reply:
This suggestion does not work since it results in a compile error: `Cannot resolve overloaded method 'setCellValue'`.